app/shared/reducers/account/actions.js

```diff
@@ -0,0 +1,38 @@
+import * as types from '../../actions/types';
+
+const initialState = {
+  lastError: {},
+  lastTransaction: {},
+  list: [],
```

Comment:
This component should be designed to support actions from multiple accounts. An array isn't going to cut it. 

It should be similar to the actual `accounts` reducer in that it stores lists of actions under each individual account.

```
actions = {
  jesta: [ ...items ],
  dafuga: [ ...items ]
}
```

You are already passing back `account_name` with the results of each action call, so it shouldn't be too hard to organize further.

Reply:
ahh that's a good point.